Unittests/LocalDeclaration/LocalDeclarationNullAnnotatorTest.cs

```diff
@@ -14,10 +14,10 @@
 using System;
 using System.Linq;
 using Microsoft.CodeAnalysis.CSharp.Syntax;
-using NullableReferenceTypesRewriter.ConsoleApplication.LocalDeclaration;
+using NullableReferenceTypesRewriter.LocalDeclaration;
 using NUnit.Framework;
 
-namespace NUnit2To3SyntaxConverter.Unittests.LocalDeclaration
+namespace NullableReferenceTypesRewriter.Unittests.LocalDeclaration
 {
   public class LocalDeclarationNullAnnotatorTest

```

Comment:
Typo in `LocalDecalrationNullAnnotator_DoesAnnotateDeclarationsCallingNullableMethods`